test/util/externalservices/MdgWcoDecService.scala

```diff
@@ -17,24 +17,46 @@
 package util.externalservices
 
 import com.github.tomakehurst.wiremock.client.WireMock._
+import com.github.tomakehurst.wiremock.matching.UrlPattern
 import play.api.test.Helpers._
-import util.{ExternalServicesConfig, CustomsDeclarationsExternalServicesConfig, WireMockRunner}
+import util.{CustomsDeclarationsExternalServicesConfig, ExternalServicesConfig, WireMockRunner}
 
 trait MdgWcoDecService extends WireMockRunner {
   private val urlV1MatchingRequestPath = urlMatching(CustomsDeclarationsExternalServicesConfig.MdgWcoDecV1ServiceContext)
   private val urlV2MatchingRequestPath = urlMatching(CustomsDeclarationsExternalServicesConfig.MdgWcoDecV2ServiceContext)
+  private val urlV3MatchingRequestPath = urlMatching(CustomsDeclarationsExternalServicesConfig.MdgWcoDecV3ServiceContext)
 
-  def startMdgWcoDecService(): Unit = {
-    setupMdgWcoDecServiceToReturn(ACCEPTED)
+
+  def startMdgWcoDecServiceV1(): Unit = {
+    setupMdgWcoDecServiceToReturn(ACCEPTED, urlV2MatchingRequestPath)
```

Comment:
Need to check why these are passing as v1 and v3 are matching against the wrong path. v2 is ok.